Formula/g/git-get.rb

```diff
@@ -0,0 +1,36 @@
+class GitGet < Formula
+  desc "Better way to clone, organize and manage multiple git repositories"
+  homepage "https://github.com/grdl/git-get"
+  url "https://github.com/grdl/git-get/archive/refs/tags/v0.6.1.tar.gz"
+  sha256 "82a24231bffacc0a8eaf50d21892ac6160f5f616cafccd2381b8bc9845452ff4"
+  license "MIT"
+  head "https://github.com/grdl/git-get.git", branch: "main"
+
+  depends_on "go" => :build
+  depends_on "go-md2man" => :build
+
+  def install
+    ldflags = "-s -w -X git-get/pkg/cfg.version=#{version}"
+    system "go", "build", *std_go_args(ldflags:, output: "git-get"), "./cmd"
+    bin.install "git-get"
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags:, output: bin/"git-get"), "./cmd"
```
Sorry, I forgot you can remove `bin.install` here

Reply:
I see, brew will always copy all build artifacts in the bin dir to the formula's cellar?